x-pack/plugins/infra/public/pages/metrics/hosts/hooks/use_host_table.ts

```diff
@@ -7,19 +7,28 @@
 
 import { last } from 'lodash';
 import { useMemo } from 'react';
-import type { SnapshotNode, SnapshotNodeMetric } from '../../../../../common/http_api';
-import { HostMetics } from '../components/hosts_table_columns';
+import { SnapshotNode } from '../../../../../common/http_api';
+import { HostMetics, HostNodeRow } from '../components/hosts_table_columns';
 
-type MappedMetrics = Record<keyof HostMetics, SnapshotNodeMetric>;
+export type MappedMetrics = Record<keyof HostMetics, number | null | undefined>;
 
 export const useHostTable = (nodes: SnapshotNode[]) => {
-  const items: MappedMetrics[] = useMemo(() => {
-    return nodes.map(({ metrics, path }) => ({
-      ...last(path),
+  const items: HostNodeRow[] = useMemo(() => {
+    const valuesMapping: Record<keyof HostMetics, 'value' | 'avg' | 'max'> = {
+      cpuCores: 'value',
+      rx: 'avg',
+      tx: 'avg',
+      memory: 'avg',
+      memoryTotal: 'avg',
+    };
+    return nodes.map(({ metrics, path, name }) => ({
+      name,
+      os: last(path)?.os ?? '-',
```

Comment:
If you prefer, you can get rid of `lodash` by doing the following: 

```
const lastPath = { ...path.at(-1), os: path.at(-1)?.os ?? '-' };
```

new TS cool feature